### PR TITLE
Version 2.1 

### PR DESCRIPTION
**Pull request by keithf4**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
+### 2.1 
+ * IMPORTANT UPGRADE NOTE FOR CRUNCHY PACKAGE USERS: In version 2.0, the Crunchy provided extras for node_exporter were split out from the pg##-extras package. A dependency was kept between these packages to make upgrading easier. For 2.1, the dependency between these packages has been removed. When upgrading from 1.7 or less, if you have node_exporter and postgres_exporter running on the same systems, ensure that you install the separate node_exporters_extras package after the update. See the README for the full package name(s).
```
Comment (jkatz) [inline CHANGELOG.md]: * **IMPORTANT UPGRADE NOTE FOR CRUNCHY DATA PACKAGE USERS**: In version 2.0, the Crunchy Data packages provided extras for `node_exporter` were split out from the `pg##-extras` package. A dependency was kept between these packages to make upgrading easier. For 2.1, the dependency between these packages has been removed. When upgrading from 1.7 or earlier, if you have `node_exporter` and `postgres_exporter` running on the same systems, ensure that you install the separate `node_exporters_extras package` after the update. See the README for the full package names.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+
```
Comment (jkatz) [inline CHANGELOG.md]: with the **bold text** above in place i'd remove this extra whitespace

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,15 @@
...
+ * Fixed Grafana data source to use the "proxy" mode instead of "direct" with default install. Should fix issues encountered with the default setup when Grafana is not installed on the same system as Prometheus.
+ * Renamed functions_pg##.sql file to setup_pg##.sql to better clarify what it's for (and because it's not just functions).
+ * Added ccp_wal_activity metric to help monitor WAL generation rate. Note that for PG96 and lower, an new security definer function must be added (can just run setup_pg##.sql again). New metric definition is located in the queries_pg##.yml file. No default rules have been added since this is very use-case dependent.
+ * Improved accuracy of Idle In Transaction monitoring times in PostgreSQL. Base the time measured on the state change of the session vs the total transaction runtime.
```
Comment (jkatz) [inline CHANGELOG.md]: * Improved accuracy of "Idle In Transaction" monitoring times in PostgreSQL. It is now based on the time measured on the state change of the session vs the total transaction runtime.

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,7 +19,7 @@
    * See full release notes for 0.16.0 - https://github.com/prometheus/node_exporter/releases/tag/v0.16.0
  * The postgres_exporter service no longer uses a symlink in /etc/sysconfig to point to a default "postgres_exporter" file. This was causing issues with several upgrade scenarios. New installation instructions now have the service pointing directly to the relevant sysconfig file for the major PostgreSQL version. 
    * IMPORTANT: If you are using the default postgres_exporter service, you will need to update your service name so it uses the proper sysconfig file. See the README file for the new default service name in the "Enable Services" section and run the "enable" command found there. You should then also disable/remove the old service so it doesn't try to start again in the future.
```
Comment (jkatz) [inline CHANGELOG.md]: **IMPORTANT**

I recommend adding the boldface

**Comment by keithf4** [comment]
I've applied the formatting requests from @jkatz . Going to close and resubmit this PR without the previous commit history getting pulled in.

**Pull request closed without merging**